### ISSUE TITLE
Failing load test Metric10kDPS/SignalFx

### ISSUE THREAD

**Issue by dmitryax** [open-telemetry/opentelemetry-collector-contrib#9735]
```
Metric10kDPS/SignalFx                   |FAIL  |      9s|    68.4|    70.6|         58|         99|    626500|        612500|RAM consumption is 99 MiB, max expected is 98 MiB
```

**Comment by djaglowski** [comment]
This is likely due to creeping memory baseload, as described and resolved by #9684

**Comment by bogdandrutu** [comment]
per @djaglowski closing this
